sdk/src/feathers/controls/TextInput.ls

```diff
@@ -133,6 +133,12 @@ package feathers.controls
          * @private
          */
         protected var _text:String = "";
+        
+        /**
+         * @private
+         * Par of a Workaround fix that specially prevents editing when not in focus
```

Comment:
@SmilyOrg Can you review this workaround and see if it should be rolled into master?